ament_cmake_test/ament_cmake_test/__init__.py

```diff
@@ -0,0 +1,351 @@
+# Copyright 2014-2015 Open Source Robotics Foundation, Inc.
```

Comment:
Currently the history of this file is lost. Please copy the content of the file without any changes in a separate commit. Then in a second commit make any necessary modifications to the original file as well as the new one. That should allow git to track the history of the content.

Reply:
@dirk-thomas I'm not seeing the history in still. . . I think maybe I have to do a 'move' operation instead of a copy.  Can you confirm that the history is still missing?

Reply:
Try looking at the blame view instead. The history view might not work correctly either way.

Reply:
Blame shows that I wrote 100% of the lines five hours ago.  I'll try a move operation followed by an edit so at least ament_cmake_test/ament_cmake_test/__init__.py will have history.  Since that's discoverable from run_test.py, I think that should be enough